camshare/camshare/PhotoAlbumViewController.swift

```diff
@@ -8,28 +8,34 @@
 
 import UIKit
 
+var selectedImageIndex: Int = 0
+
 class PhotoAlbumViewController: ViewController {
     // MARK: OUTLETS
     @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var imageView: UIImageView!
+    
 
     // MARK: Properties
-    var images = [UIImage(named: "image-1"), UIImage(named: "Image2"), UIImage(named: "Image3"),
-    UIImage(named: "Image4"), UIImage(named: "Image5"), UIImage(named: "Image6"),
-    UIImage(named: "Image7"), UIImage(named: "Image8"), UIImage(named: "Image9"),
-    UIImage(named: "Image10"), UIImage(named: "Image11"), UIImage(named: "Image12")]
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        //navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Add", style: .plain, target: self, action: #selector(addTapped))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .stop, target: self, action: #selector(addTapped))
 
         // Do any additional setup after loading the view.
     }
+    @objc func addTapped() {
+        performSegue(withIdentifier: "Logout", sender: self)
+        print("App will now logout")
```

Comment:
Dont leave prints in your code its bad practice to leave any kind of logging that could potentially leak information or behavior of your app. That should only be for you when debugging

Reply:
Oky cool. I also didn't delete my comments yet, I forgot about it 🙈